src/getters.ts

```diff
@@ -0,0 +1,10 @@
+import { isIssue, ValueProcessor } from './types';
+import { ValidationError } from './validation-error';
+
+export const base = <T>(check: ValueProcessor<T>, value: () => unknown, nest?: string | number): T => {
```

Comment:
Could we please name this something more meaningful. Maybe `validateOrThrow` or something like that